### DISCUSSION TITLE
5.2.6 on Android 8.1.0 with libsslocal exits too fast (exit code: 135) error

### DISCUSSION THREAD

**Discussion by Suitear** [shadowsocks/shadowsocks-android#2985] [Q&A]
Please read contributing guidelines. Thanks.

**Describe the bug**
Update shadowsocks on a MotoG device(Android 8.1.0) to 5.2.6

**To Reproduce**
1. Just click the switch in shadowsocks.

**Expected behavior**

**Screenshots**
![05537AD4-3E11-4852-A256-61B4274056F0](https://user-images.githubusercontent.com/9960618/151735071-225106c5-b516-4842-bdf4-c770b32f77f7.jpeg)

**Smartphone (please complete the following information):**
 - Android version: [8.1.0 OPSS28.85-17-6]
 - Device: [MotoG Amazon version]
 - Version: 
 

**Configuration**
Put an `x` inside the [ ] that applies.

* [x] IPv4 server address
* [ ] IPv6 server address
* [x ] Client IPv4 availability
* [ ] Client IPv6 availability
* Encrypt method: xchacha20-ietf-poly1305
* Route
  * [ ] All
  * [ ] Bypass LAN
  * [ ] Bypass China
  * [x] Bypass LAN & China
  * [ ] GFW List
  * [ ] China List
  * [ ] Custom rules
* [ ] IPv6 route
* [ ] Apps VPN mode
  * [ ] Bypass mode
* Remote DNS: 8.8.8.8
* [ ] DNS over UDP
* Plugin configuration (if applicable):
* [ ] Auto Connect
* [ ] TCP Fast Open
* If you're not using VPN mode, please supply more details here:

**Additional context**

**Comment by Jishun** [comment]
I've encountered libsslocal exists too fast error after upgraded to the latest version as of now (5.3.3) for using the latest crypto method `2022-blake3-aes-256-gcm`
in my case This error was caused by a password that did not meet the requirements, if you have setup the server side, you'll see the actual error complaining about the password and exits when starting. 

this crypto requires password of a `32 bytes string` AND `base64 encoded` , same value in both client config and server config, 
by changing the password to meet the requirements, my tunnel was setup and usable

if you see this error, may first try a different crypto method to see if it's gone then you know it's this method specific error and likely to be the password issue if so

hope that helps.
